_functions_exo_docker.sh

```diff
@@ -0,0 +1,210 @@
+  echo_info "Exo docker container ${DEPLOYMENT_EXO_DOCKER_CONTAINER_NAME} stopped."
+}
+
+do_start_exo_docker() {
+  local DOCKER_ARGS=""
+  local DATABASE_TYPE=""
+  DEPLOYMENT_LOG_PATH="${DEPLOYMENT_DIR}/logs/exo"
+  if [ -e "${DEPLOYMENT_LOG_PATH}/platform.log" ]; then
+      rm -rf ${DEPLOYMENT_LOG_PATH}/platform.log
+  fi
+  echo_info "Starting Exo docker..."
+  if ! ${DEPLOY_EXO_DOCKER}; then
+    echo_info "Exo wasn't deployed in docker, skiping exo docker startup"
+    return
+  fi
+  mkdir -p ${DEPLOYMENT_DIR}/logs/exo
+  echo_info "Starting exo docker container ${DEPLOYMENT_EXO_DOCKER_CONTAINER_NAME} based on image ${DEPLOYMENT_EXO_DOCKER_IMAGE}:${DEPLOYMENT_EXO_DOCKER_IMAGE_VERSION}"
+  # Ensure there is no container with the same name
+  delete_docker_container ${DEPLOYMENT_EXO_DOCKER_CONTAINER_NAME}  
+  
+  case "${DEPLOYMENT_DATABASE_TYPE}" in      
+      DOCKER_MYSQL)
+        DATABASE_TYPE="mysql"
+      ;;      
+      DOCKER_POSTGRES)
+        DATABASE_TYPE="pgsql"
+      ;;
+      *)
+        echo_error "${DEPLOYMENT_DATABASE_TYPE} not supported in eXoDocker"
+        exit 1
+      ;;
+   esac
+  
+  DOCKER_ARGS="${DOCKER_ARGS} run -d"
+  DOCKER_ARGS="${DOCKER_ARGS} -p ${DEPLOYMENT_HTTP_PORT}:8080"
+  DOCKER_ARGS="${DOCKER_ARGS} -p ${DEPLOYMENT_AJP_PORT}:8443"
+  DOCKER_ARGS="${DOCKER_ARGS} -p ${DEPLOYMENT_RMI_REG_PORT}:10001" 
+  DOCKER_ARGS="${DOCKER_ARGS} -p ${DEPLOYMENT_RMI_SRV_PORT}:10002" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_REGISTRATION=false" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_ADDONS_LIST=${DEPLOYMENT_ADDONS}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_ADDONS_REMOVE_LIST=${DEPLOYMENT_ADDONS_TOREMOVE}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_ADDONS_CATALOG_URL=${DEPLOYMENT_ADDONS_CATALOG}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_PATCHES_CATALOG_URL=${DEPLOYMENT_PATCHES_CATALOG}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_PATCHES_LIST=${DEPLOYMENT_PATCHES}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_JVM_SIZE_MAX=${DEPLOYMENT_JVM_SIZE_MAX}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_JVM_SIZE_MIN=${DEPLOYMENT_JVM_SIZE_MIN}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_PROXY_VHOST=${DEPLOYMENT_APACHE_VHOST_ALIAS}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_DATA_DIR=/srv/exo" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_DB_TYPE=${DATABASE_TYPE}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_DB_NAME=${DEPLOYMENT_DATABASE_NAME}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_DB_USER=${DEPLOYMENT_DATABASE_USER}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_DB_PASSWORD=${DEPLOYMENT_DATABASE_USER}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_MONGO_HOST=${DEPLOYMENT_CHAT_MONGODB_HOSTNAME}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_MONGO_PORT=${DEPLOYMENT_CHAT_MONGODB_PORT}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_MONGO_DB_NAME=${DEPLOYMENT_CHAT_MONGODB_NAME}" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_ES_EMBEDDED=${DEPLOYMENT_ES_EMBEDDED}"   
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_MAIL_FROM=noreply+acceptance@exoplatform.com" 
+  DOCKER_ARGS="${DOCKER_ARGS} -e EXO_MAIL_SMTP_HOST=localhost" 
```

Comment:
This will not work inside the container.
Perhaps it the right moment to add a side mailhog container